### PR TITLE
Enable tracking of external links in GA

### PR DESCRIPTION
**Pull request by davidpmccormick**
## Type
✨ Feature  

## Value
Will allow us to see what the destination URL is from an exit page in GA.

## Screenshot
🔍

### All
- [x] Demoed to the relevant people
- [x] PR labelled and assigned
- [x] Any introduced code complexity has been flagged

### CODE REVIEW

File: client/js/tracking.js [hunk 1]
```diff
@@ -15,3 +17,29 @@ export default {
     });
   }
 };
+
+function trackOutboundLink(url) {
```
Comment (davidpmccormick) [inline client/js/tracking.js]: As per https://support.google.com/analytics/answer/1136920?hl=en

File: client/js/tracking.js [hunk 2]
```diff
@@ -15,3 +17,29 @@ export default {
...
+    }
+  });
+};
+
+function getDomain(url) {
+  return url.replace('http://', '').replace('https://', '').split('/')[0];
+}
+
+function isExternal(url) {
+  return getDomain(document.location.href) !== getDomain(url);
```
Comment (jamesgorrie) [inline client/js/tracking.js]: What about hash links - are these going to be considered external now?

Comment (davidpmccormick) [inline client/js/tracking.js]: Good point. Should we choose what to ignore here (like mailto: and hash links) or do more of an opt in thing – wrapping things that we definitely want to track in e.g. a 'data-track-outboud' block?

Comment (davidpmccormick) [inline client/js/tracking.js]: > What about hash links

…on closer inspection, all of our supported browsers will report the full domain, not just the hash fragment, so they won't be considered to be external (assuming they're hash links on our domain).

**Pull request merged**